doc/python/interconnectivity.rst

```diff
@@ -14,11 +14,11 @@ Interconnectivity
 
     .. attribute:: source
 
-        The source end point of the connection (type: :class:`arbor.cell_member`).
+        The source end point of the connection (type: :class:`arbor.cell_member` or a (gid, index) tuple).
```

Comment:
I don't know whether this is necessary here. The `(gid, index)` tuple **is** a `arbor.cell_member`, it's automatically converted into one. Maybe this can go into the documentation of the `arbor.cell_member` class.